src/JSTest.TestAdapter/TestExecutor.cs

```diff
@@ -49,6 +49,12 @@ public void RunTests(IEnumerable<TestCase> tests, IRunContext runContext, IFrame
             catch (JSTestException e)
             {
                 frameworkHandle.SendMessage(TestMessageLevel.Error, e.ToString());
+
+                if (e.InnerException is TimeoutException)
+                {

```

Comment:
@karanjitsingh why don't we dispose off the testrunner in case of any other exception?

Reply:
We don't know if any other exception comes with node process stopping. But yes we're definitely missing some cases here.

Reply:
Done.